src/pages/Navigation/Users/User/User.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Link, useLocation } from 'react-router-dom';
+
+// Components
+import UserImg from '../../../../components/UserImg/UserImg';
+
+// Routes
+import { PROFILE_USER_WITHOUT_ID } from '../../../../consts/routes';
+
+// Styles
+import classes from './User.module.scss';
+
+// Add avatar
+function User({ id, username, avatar }) {
+  const location = useLocation();
+  return (
+    <li className={classes.User}>
+      <Link
+        to={{
+          pathname: `${PROFILE_USER_WITHOUT_ID}${id}`,
+          state: { from: location.pathname },
```

Comment:
what is the purpose of storing origination path in route state? I saw only that you use it to mimic browser back button - there are built in methods for that